extensions/wikia/PortabilityDashboard/PortabilityDashboardModel.class.php

```diff
@@ -150,6 +150,6 @@ private function connect( $type = DB_SLAVE ) {
 	 * @return int
 	 */
 	private function floatToPercent( $float ) {
-		return round( $float * 100 );
+		return round( $float * 100, 2 );
```

Comment:
I wonder if we need to format this rather like sprintf("%.2f", $float*100) to make sure we format the number consistently